src/test/src/lib/validation/nationality.test.ts

```diff
@@ -0,0 +1,44 @@
+import nationalityValidator from "../../../../../lib/utils/error_manifests/nationality";
+import { validationResult } from "express-validator";
+import { Request } from "express";
+
+// Mock the express Request object for testing purposes
+const mockRequest: Request = {} as Request;
+
+describe("nationalityValidator", () => {
```

Comment:
Can you add few more tests to cover other validation scenarios ? Also noticed there are no tests for controller.

Reply:
Agreed! You would probably want to test negative scenarios as well. What happens if I pass "In" as a string,
what happens in case of blank string etc.